milli/src/search/new/vector_sort.rs

```diff
@@ -49,16 +52,15 @@ impl<Q: RankingRuleQueryTrait> VectorSort<Q> {
         ctx: &mut SearchContext<'_>,
         vector_candidates: &RoaringBitmap,
     ) -> Result<()> {
-        let readers: std::result::Result<Vec<_>, _> =
-            ctx.index.arroy_readers(ctx.txn, self.embedder_index).collect();
-        let readers = readers?;
+        let readers: Result<Vec<_>> =
+            ctx.index.arroy_readers(ctx.txn, self.embedder_index, self.quantized).collect();
 
         let target = &self.target;
         let mut results = Vec::new();
 
-        for reader in readers.iter() {
+        for reader in readers?.iter() {
             let nns_by_vector =
-                reader.nns_by_vector(ctx.txn, target, self.limit, None, Some(vector_candidates))?;
+                reader.nns_by_vector(ctx.txn, target, self.limit, Some(vector_candidates))?;
```

Comment:
This none was to control the number of node to inspect in arroy, and a second None appeared after the binary quantization to control how much overflow we want to do.
Since Meilisearch will never use any of these two options, I got rids of it in my `ArroyWrapper`.
Also, in the near future I would like to rewrite the way we make query in arroy because there is way too many options here 🥶